indexer/pkg/config/config.go

```diff
@@ -200,14 +146,16 @@ type AggregatorReaderConfig struct {
 	Address string `toml:"Address"`
 	// Since is the unix timestamp in seconds to start reading from.
 	Since int64 `toml:"Since"`
+	// APIKey is the client's API Key (UUID format)
+	APIKey string `toml:"APIKey"`
+	// Secret is the HMAC secret used to sign requests
+	Secret string `toml:"Secret"`
```

Comment:
I think those will eventually need to be env variable. I had to change these to be env variable in aggregator for staging deployment